provider/azure/networking.go

```diff
@@ -216,26 +215,3 @@ func nextSecurityRulePriority(group network.SecurityGroup, min, max int32) (int3
 		"no priorities available in the range [%d, %d]", min, max,
 	)
 }
-
```

Comment:
moved to upgrades.go below as this is only used there and want to keep networking.go free of such upgrade related methods